rnaseq/bam_to_trimmed_fastq_and_hisat_alignments.cwl

```diff
@@ -2,58 +2,68 @@
 
 cwlVersion: v1.0
 class: Workflow
-label: "alignment"
+label: "bam to trimmed fastqs and HISAT alignments"
 requirements:
+    - class: MultipleInputFeatureRequirement
     - class: SubworkflowFeatureRequirement
+    - class: InlineJavascriptRequirement
 inputs:
-    reference_index:
-        type: File
-        secondaryFiles: [".1.ht2", ".2.ht2", ".3.ht2", ".4.ht2", ".5.ht2", ".6.ht2", ".7.ht2", ".8.ht2"]
-    instrument_data_bam:
+    bam:
         type: File
-    read_group_id:
-        type: string
-    read_group_fields:
-        type: string[]
-    trimming_adapters:
+    adapters:
         type: File
-    trimming_adapter_trim_end:
+    adapter_trim_end:
         type: string
-    trimming_adapter_min_overlap:
+    adapter_min_overlap:
         type: int
-    trimming_max_uncalled:
+    max_uncalled:
         type: int
-    trimming_min_readlength:
+    min_readlength:
         type: int
+    read_group_id:
+        type: string
+    read_group_fields:
+        type:
+            type: array
+            items: string
+    reference_index:
+        type: File
```

Comment:
Was removing the `secondaryFiles` here intentional?  They're still present in `hisat2_align.cwl`.  (I guess they aren't required to be repeated if this was still working!)